stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -150,18 +167,43 @@ private class TraceContext {
                 outputs: [CTensorHandle]) {
     internalConsistencyCheck(traceGraphFn == nil)
     var symbolicOutputs: [TF_Output] = []
+    // Only add symbolic output tensors as the outputs of the trace graph function.
+    // For example, let the tracee be:
+    //   func foo(x: Tensor) -> (Tensor, Tensor) {
+    //     let y = Tensor<Float>(1.0)
+    //     return (x + x, y)
```

Comment:
The test added here does not exercise this scenario?

Reply:
Discussed in person, and `TracerTests.testAllBackends("Basic_IntermediateTensors")` should cover this case.